code/game/machinery/doors/airlock.dm

```diff
@@ -627,9 +627,20 @@
 		. += span_notice("Alt-click [src] to [ secondsElectrified ? "un-electrify" : "permanently electrify"] it.")
 		. += span_notice("Ctrl-Shift-click [src] to [ emergency ? "disable" : "enable"] emergency access.")
 
-/obj/machinery/door/airlock/add_context(atom/source, list/context, obj/item/held_item, mob/living/user)
+/obj/machinery/door/airlock/add_context(atom/source, list/context, obj/item/held_item, mob/user)
 	. = ..()
 
+	if(isaicamera(user) || issilicon(user))
+		var/ui = SStgui.try_update_ui(user, src)
+		if(!ui)
+			context[SCREENTIP_CONTEXT_LMB] = "Open UI"
+		if(!(machine_stat & BROKEN))
+			context[SCREENTIP_CONTEXT_SHIFT_LMB] = density ? "Open" : "Close"
+			context[SCREENTIP_CONTEXT_CTRL_LMB] = locked ? "Unbolt" : "Bolt"
+			context[SCREENTIP_CONTEXT_ALT_LMB] = isElectrified() ? "Unelectrify" : "Electrify"
+			context[SCREENTIP_CONTEXT_CTRL_SHIFT_LMB] = emergency ? "Unset EA" : "Set EA"
```

Comment:
I'm guessing you've already tried "emergency access" and it was too long, but the abbreviation here does make me sad, since new players won't know what it means. ☹️

Reply:
Not that I have any better answers here, if anyone does reply here

Reply:
![Space Station 13 2022-02-18 170425](https://user-images.githubusercontent.com/25089914/154615564-6fe4c77a-dcca-4e7c-a6d2-947453b85ff2.png)

![Space Station 13 2022-02-18 170430](https://user-images.githubusercontent.com/25089914/154615530-c967cedb-aa18-4cd9-893f-8e06ad3240ec.png)

Think it's probably fine actually
️